services/jupyter/src/pyproject.toml

```diff
@@ -26,6 +26,12 @@ transformers = "^4.22.2"
 typer = "^0.6.1"
 wandb = "^0.13.3"
 boto3 = {version = "^1.24.89", source = "pypi"}
+rubrix = {extras = ["listeners"], version = "^0.18.0"}
+loguru = "^0.6.0"
+snorkel = {version = "^0.9.9", source = "pypi"}
+modAL = "^0.4.1"
+skorch = "^0.12.0"
```

Comment:
are these needed? If we just use `small-text` they can go I think

Reply:
Diego created this branch and I work on top of it so I could use the setup. I guess we don't need that.

Reply:
yeah, I installed them to make some notebooks work. If we are not going to use them it's safe to remove them.